update-js.sh

```diff
@@ -4,7 +4,7 @@ rm -f yarn.lock
 yarn add @fontsource/open-sans swagger-ui-dist
 yarn add --production es6-promise fetch react react-dom graphiql graphql-playground-react@1.7.26 redoc
 
-dest=src/Bridge/Symfony/Bundle/Resources/public/fonts/open-sans/
```

Comment:
This is probably not wanted. cc @soyuka

Reply:
np for now I'll move this later

Reply:
It's required, since everything has been moved